tests/test_basic.py

```diff
@@ -585,19 +587,46 @@ def test_switch_device_identifier(
 
     dm = runtime.device_manager()
     p.video[0].storage.identifier = dm.select(DeviceKind.Storage, "tiff")
-    p.video[0].storage.settings.filename = f"{request.node.name}.tif"
+    p.video[0].storage.settings.uri = f"{request.node.name}.tif"
     p = runtime.set_configuration(p)
     assert p.video[0].storage.identifier.name == "tiff"
 
     runtime.start()
     runtime.stop()
 
     # will raise an exception if the file doesn't exist or is invalid
-    with tifffile.TiffFile(p.video[0].storage.settings.filename):
+    with tifffile.TiffFile(p.video[0].storage.settings.uri):
         pass
 
     # cleanup
-    os.remove(p.video[0].storage.settings.filename)
+    os.remove(p.video[0].storage.settings.uri)
+
+
+def test_acquire_unaligned(runtime: Runtime):
+    dm = runtime.device_manager()
+    props = runtime.get_configuration()
+    props.video[0].camera.identifier = dm.select(
+        acquire.DeviceKind.Camera, ".*empty.*"
+    )
+
+    # sizeof(VideoFrame) + 33 * 47 is not divisible by 8
```

Comment:
What are the implications of it not being divisible by 8? Is there an opportunity to add context to this comment?

Reply:
I'll add a docstring to this test, because it's not clear from the test itself.